Saas/Get-BcEnvironments.ps1

```diff
@@ -1,43 +1,72 @@
-﻿<# 
+﻿<#
  .Synopsis
-  Function for retrieving a list of environments from an online Business Central tenant
+  Function for retrieving a list of all environments or one selected environment from an online Business Central tenant
  .Description
-  Function for retrieving a list of environments from an online Business Central tenant
-  Wrapper for https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/administration/administration-center-api#get-environments-and-get-environments-by-application-family
+  Function for retrieving a list of all environments or one selected environment from an online Business Central tenant
+  Wrapper for https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/administration/administration-center-api_environments#get-environment-by-application-family-and-name
  .Parameter bcAuthContext
   Authorization Context created by New-BcAuthContext.
  .Parameter applicationFamily
   Application Family in which the environment is located. Default is BusinessCentral.
+ .Parameter environment
+  Name of the environment
+ .Parameter apiVersion
+  API version. Default is 2.15.
  .Example
   $authContext = New-BcAuthContext -includeDeviceLogin
   Get-BcEnvironments -bcAuthContext $authContext
+ .Example
+  $authContext = New-BcAuthContext -includeDeviceLogin
+  Get-BcEnvironments -bcAuthContext $authContext -environment "MySandbox"
 #>
 function Get-BcEnvironments {
     Param(
-        [Parameter(Mandatory=$true)]
+        [Parameter(Mandatory = $true)]
         [Hashtable] $bcAuthContext,
-        [string] $applicationFamily = "BusinessCentral"
+        [string] $applicationFamily = 'BusinessCentral',
+        [string] $environment,
+        [string] $apiVersion = "2.15"

```

Comment:
I think we should have the apiVersion be "v2.15" instead of 2.15 - and then use the full string when creating the URL - all over.

Reply:
Do you prefer whole string like "v2.15"? If yeas I should do it same way in other functions right?

Reply:
Done

Reply:
yes - all functions.
Some times there are versions like beta and alpha, so preceding a 'v' wouldn't work.